GameWindow.java

```diff
@@ -38,25 +38,30 @@
 	public final String TITLE = "Melee Tech Trainer";
 
 	private static SaveFile hs;
-
+	
 	private boolean running = false;
 	private Thread thread;
 
 	// sound files
 	private File aSound = new File("./Notes/A.wav");
 	private File cSound = new File("./Notes/C.wav");
-
+	private File eSound = new File("./Notes/E.wav");
+	private File gsSound = new File("./Notes/Gsharp.wav");

```

Comment:
As a general rule if you have a whole ton of sounds you don't want to include each as an individual variable like this. As I'm sure you've noticed, this makes accessing them a bit annoying since you need to know the variable name of the sound you want, which generally leads to a giant if statement to pick the right variable name.

You can make this drastically simpler by using a `Map`, which is sort of like an array except that the indices can be anything you want rather than always being consecutive natural numbers.

```java
// imports for your package
import java.util.Map;
import java.util.HashMap;

// The rest of the code below goes inside your class...

// where you define your sounds. I've made this static and final since these are just constants
private static final Map<String, File> sounds = new HashMap<>();

// Initialize the map
static {
  // The first argument ("E") is called the key, and the second argument (the file) is called the "value"
  sounds.put("E", new File("./Notes/E.wav");
  sounds.put("G#", new File("./Notes/Gsharp.wav");
  //and so forth for your other notes
}

// Now you can access a file given its map key somewhere in your code
sounds.get("E");
```

It's typically better to use `enum`s instead of `String`s for this sort of thing (keys in `Map`s), but that's a bit more advanced so I wouldn't worry about it for now.